cache-dependency-binary/index.js

```diff
@@ -19,6 +19,18 @@ async function handleRequest(request) {
   let owner, toolName, version, target
   [_, owner, toolName, version, target] = urlParts
 
+  if (target === 'aarch64-apple-darwin') {
+    const asset = `${toolName}-${version}-${target}.tar.gz`
```

Comment:
```suggestion
    const asset = `${toolName}-v${version}-${target}.tar.gz`
```

And move the if statement back to where it was

requires a re-upload to KV, but this will fix the behavior difference